docker-compose.yml

```diff
@@ -59,16 +60,17 @@ services:
       - POSTGRES_USER=kinetic
       - POSTGRES_PASSWORD=kinetic
       - POSTGRES_DB=kinetic_test
-  petstore:
-    image: "swaggerapi/petstore3"
-    networks:
-      - kinetic
-    environment:
-      - SWAGGER_HOST=http://0.0.0.0:4008
-      - SWAGGER_BASE_PATH=/api/v1
-      - SWAGGER_URL=http://0.0.0.0:4006
-    ports:
-      - "4008:8080"
+#  TODO: Investigate if needed
```

Comment:
Wasn't sure if anything was necessarily using this, tried looking through the code but couldn't find anything. It has a slow startup/shutdown time with docker compose and we get some warnings about something with the container image 🤔